client/src/pages/Login/Login.tsx

```diff
@@ -37,12 +34,11 @@ export default function Login(): JSX.Element {
   };
 
   return (
-    <Grid container component="main" className={classes.root}>
+    <Grid container component="main">
       <CssBaseline />
```

Comment:
This is a part of the starter code but please remove it as the 'CssBaseline' is a global reset and should be included only once in the App.tsx :)